resources/5/ajax.html

```diff
@@ -0,0 +1,83 @@
+            </header>
+            <nav>
+                <ul>
+                    <li><a href="../../index.html">Főoldal</a></li>
+                    <li><a href="../2/tablazat.html">Táblázat</a></li>
+                    <li><a href="../3/html5.html">HTML5</a></li>
+                    <li><a href="../4/chartjs.html">ChartJS</a></li>
+                    <li><a href="ajax.html" class="active">AJAX</a></li>
+                    <li><a href="../oojs.html">OOJS</a></li>
+                </ul>
+            </nav>
+            <aside>
+                <h1>AJAX alkalmazás</h1>
+                <hr>
+                <h2>Az 5. feladat megvalósítása</h2>
+            </aside>
+            <section>
+                <article>
+                <h1>AJAX alkalmazás</h1>
+                </article>
+                <div class="container">
+                    <div class="box">
+                        <p id="code">code= VE22CSabc123</p>
+                        <h3>Adatok lekérése</h3>
+                        <button id="readBtn">Adatok lekérése</button>
+                        <div id="readDiv"></div>
+                    </div>
+                    <div class="box">
+                        <h3>Magasság Statisztika</h3>
+                        <p>Összeg: <span id="sumHeight">-</span></p>
+                        <p>Átlag: <span id="avgHeight">-</span></p>
+                        <p>Legnagyobb: <span id="maxHeight">-</span></p>
+                    </div>
+                    <div class="box">
+                        <h3>Új adat létrehozása</h3>
+                        Név (kötelező, max 30):<br><input id="name1" type="text"></input><br>
+                        Magasság (cm) (kötelező, max 30): <input id="height1" type="text"></input><br>
+                        Súly (kg) (kötelező, max 30): <input id="weight1" type="text"></input><br>
+                        <button type="submit" onclick="create();">Létrehoz</button><br>
+                        <div id="createDiv"></div>
+                        <div id='createResult'></div>
+                    </div>
+                    <div class="box">
+                        <h3>Update</h3>
+                        ID (kötelező): <input id="idUpd" type="text"></input><br>
+                        <button type="submit" onclick="getDataForId();">ID adat lekérése</button><br>
+                        Név (kötelező, max 30):<br><input id="name2" type="text"></input><br>
+                        Magasság (kötelező, max 30):<br><input id="height2" type="text"></input><br>
+                        Súly (kötelező, max 30):<br><input id="weight2" type="text"></input><br>
+                        <button type="submit" onclick="update();">Frissítés</button><br>
+                        <div id='updateResult'></div>
+                    </div>
+                    <div class="box">
+                        <h3>Adat törlése</h3>
+                        ID (kötelező): <input id="idDel" type="text"></input><br>
```

Comment:
Can you please rename "idDel" to "idToDeleteInput", update the id in the javascript as well!